projects/admin.py

```diff
@@ -1,6 +1,13 @@
 from django.contrib import admin
 
-from projects.models import Invitation, Publication
+from projects.models import Invitation, Publication, Funding
+
+
+def _get_charge_code(obj):
```

Comment:
It would help to indicate what sort of object `obj` is, since this is in `projects/`, I at first assumed it is to get a project charge code.

Reply:
Maybe just a comment would suffice